var/spack/repos/builtin/packages/upp/package.py

```diff
@@ -15,10 +15,11 @@ class Upp(CMakePackage):
 
     homepage = "https://github.com/NOAA-EMC/UPP"
     git = "https://github.com/NOAA-EMC/UPP.git"
-    url = "https://github.com/NOAA-EMC/UPP/archive/refs/tags/upp_v10.0.10.tar.gz"
+    url = "https://github.com/NOAA-EMC/UPP/archive/refs/tags/upp_v8.2.0.tar.gz"
 
     maintainers = ["kgerheiser", "edwardhartnett", "Hang-Lei-NOAA"]
 
+    version("8.2.0", sha256="38de2178dc79420f42aa3fb8b85796fc49d43d66f90e5276e47ab50c282627ac")
```

Comment:
Versions are ordered newest to oldest, please move this to below 10.0.9

Reply:
solved